entity-service-api/src/main/proto/org/hypertrace/entity/query/service/v1/entity_query_request.proto

```diff
@@ -15,6 +15,8 @@ message EntityQueryRequest {
   repeated OrderByExpression orderBy = 7;
   int32 limit = 5;
   int32 offset = 6;
+  repeated GroupByExpression groupBy = 8;
+  // TODO: Do we need to support filtering on grouped values (HAVING clause)?
```

Comment:
Do we have a use-case for `HAVING` clause, @skjindal93 ?

Reply:
We always have a use-case. 

But, we don't support HAVING clause in query-service yet too. What does it take to add `HAVING` clause support in entity-service?

Reply:
Well, the attribute names used in the filter has to be consumed directly (instead of getting the sub-doc path).
So, it is quite some work to implement and test. My opinion is that if there is a strong requirement, we can implement now. Otherwise, we can defer.

Note: document-store already supports `HAVING` clause.

Reply:
Let's defer definitely. We might have to enhance `FilterConverter` later based on normal vs aggregation filtering

Reply:
Sure.